code/modules/customitems/item_defines.dm

```diff
@@ -202,37 +202,36 @@
 
 	to_chat(user, "<span class='warning'>You can't modify [target]!</span>")
 
+// Custom zippo lighters
+/obj/item/lighter/zippo/fluff
+	name = "custom zippo"
+	desc = "A custom made zippo lighter."
+	// HUD icons from custom_items
+	icon = 'icons/obj/custom_items.dmi'
```

Comment:
```suggestion
	desc = "A custom made zippo lighter."
	icon = 'icons/obj/custom_items.dmi'
```
That comment isnt necessary and we normally call it the item icon.
HUD icon makes it sound like its a sprite belonging to the icons you see while wearing HUD glasses.

Reply:
Yeah I agree with you there. Made that change in ad1b5fb to reduce duplication, but the comment isn't necessary elsewhere and my word-fu isn't quite correct (as HUD and in-world items use the same icon)